python/pyspark/ml/regression.py

```diff
@@ -1305,6 +1305,9 @@ class GeneralizedLinearRegression(JavaEstimator, HasLabelCol, HasFeaturesCol, Ha
 
     * "gamma"    -> "inverse", "identity", "log"
 
+    * "tweedie"  -> power link function specified through "linkPower". \
+                    The default link power in the tweedie family is 1 - variancePower.
```

Comment:
It will produce a model according to the specified ```variancePower``` and ```linkPower```. The doc here is to explain the value of ```linkPower``` if users don't specify.